association-frontend/helpers/graphql.ts

```diff
@@ -0,0 +1,5 @@
+import { createClient } from "urql";
+
+export const client = createClient({
+  url: "http://localhost:3020/graphql",
+});
```

Comment:
what is 3020?

also reminder: maybe after merging, we should make this URL dynamic

Reply:
ah I see now, you use the proxy setup so that's the frontend port, in that case you can just do `/graphql`

Reply:
thinking about this, as we will do the `/graphql` proxy on production as well, we won't need to change anything

```suggestion
export const client = createClient({
  url: "/graphql",
});
```